src/views/UserDetail/UserDetail.js

```diff
@@ -431,6 +505,8 @@ class UserDetail extends React.Component {
     } = this.state;
 
     const user = this.getUser();
+    const username = getFullName(user);
```

Comment:
This is petty, but given that "username" has a specific meaning in this context (i.e. `username` is an attribute of a user) and that's not the value being stored, it's a poor choice for this variable.

Reply:
I did rename `username` to `fullNameOfUser`